.gitignore

```diff
@@ -1,3 +1,5 @@
 log/
 tmp/
 .powenv
+.rspec
```

Comment:
I agree it shouldn't be ignored. If you want a different set up you can set an rpsec (global) for your personal computer that will override this one
http://stackoverflow.com/questions/1819614/how-do-i-globally-configure-rspec-to-keep-the-color-and-format-specdoc-o